social_core/__init__.py

```diff
@@ -1 +1 @@
-__version__ = '3.1.0'
+__version__ = '3.1.1'
```

Comment:
Not a big deal, but you should leave library versioning to the package maintainer.  They may want to include something that's a breaking change and then no 3.1.1 will ever be released, or they may release a 3.1.1 before ever merging your PR.